packages/react/src/components/TableCard/TableCard.jsx

```diff
@@ -200,18 +200,17 @@ const TableCard = ({
     id,
   ]);
 
-  const renderActionCell = (cellItem) => {
-    const actionList = JSON.parse(cellItem.value);
```

Comment:
Where the original actions array was being re-parsed after being stringified below. Since I removed that I destructured the needed elements in the function.